spaces/domains.json

```
@@ -51,5 +51,6 @@
   "vote.frontier.xyz": "frontier",
   "vote.oceanprotocol.com": "oceandao",
   "vote.sav3.org": "sav3",
-  "vote.zlot.finance": "zlot"
+  "vote.zlot.finance": "zlot",
+  "vote.finnexus.io": "fnx"
```

This value should be your space directory name.
```suggestion
  "vote.finnexus.io": "finnexus"
```

---

This pull request is merged ,right? 
And the modification take affect now?